spec/models/evernote_note_spec.rb

```diff
@@ -61,4 +61,20 @@
     before { @evernote_note.max_out_attempts }
     its(:attempts) { should >=  ENV['attempts'].to_i }
   end
+
+  describe '#add_task' do
+    ENV['evernote_notebooks'] = 'MYNOTEBOOK,OTHERNOTEBOOK'
+    context 'when note is in required notebook' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.